stories/17-Select.mdx

```diff
@@ -17,8 +17,33 @@ There's nothing here yet.
 
 There's nothing here yet.
 
+
+
+## Usage Guidelines
+### Form field error
+
+* Any errors should use error color from the color pallet. 
+* Highlight input field in error color, place error text below the field:
+
+   ![form field error](images/select_form_field_error.png) 
+ 
+* If the error is a missing required field, the text should be "<fieldname> is required".
```

Comment:
Needs to be escaped:
`* If the error is a missing required field, the text should be "&lt;fieldname&gt; is required".`
or using backticks:
```* If the error is a missing required field, the text should be `<fieldname> is required`.```